db/changes.go

```diff
@@ -1217,7 +1241,12 @@ func (db *DatabaseCollectionWithUser) getLateFeed(feedHandler *lateSequenceFeed,
 				Seq: logEntry.Sequence,
 			}
 			change := makeChangeEntry(logEntry, seqID, singleChannelCache.ChannelID())
-			feed <- &change
+			select {
+			case <-ctx.Done():
+				return
+
+			case feed <- &change:
+			}
```

Comment:
As elsewhere, ctx.Err() instead?

Reply:
See above on filing up the buffer and cancelling while the buffer is full.